packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -75,37 +75,52 @@ class DateTimeUtils {
    * Synchronize only the time for a given moment object.
    * @param {object} momentDate - The moment object to synchronize the date and time.
    * @param {string} time - The time to synchronize with the moment object.
+   * @param {boolean} seconds - If true seconds will be processed
    * @return {object} - The synchronized moment object.
    */
-  static updateTime(momentDate, time) {
+  static updateTime(momentDate, time, seconds) {
     if (!momentDate || !momentDate.isValid()) {
       return null;
     }
 
     const newDate = momentDate.clone();
-    const date = moment(time, 'HH:mm', true);
+    const date = moment(time, seconds ? 'HH:mm:ss' : 'HH:mm', true);
+
+    if (seconds) {
+      return newDate.hour(date.get('hour')).minute(date.get('minute')).second(date.get('second'));
+    }
 
     return newDate.hour(date.get('hour')).minute(date.get('minute'));
   }
 
+  /**
+   * Gets the time from a date and time
+   * @param {string} time An ISO 8601 string to get the time of
+   * @param {boolean} second Whether or not seconds are enabled
+   * @return {string} The time from the date and time string
+   */
+  static getTime(time, second) {
+    return DateUtil.formatISODate(time, second ? 'HH:mm:ss' : 'HH:mm');
+  }
+
   /**
    * Determines if the date and time are valid and conform to the given format.
    * @param {string} date - The date to validate.
    * @param {string} time - The time to validate.
    * @param {string} format - The date/time format to use for the validation.
    * @return {boolean} - True if both the date and time are valid and conform to the format.
    */
-  static isValidDateTime(date, time, format) {
-    return DateUtil.isValidDate(date, format) && DateTimeUtils.isValidTime(time);
+  static isValidDateTime(date, time, format, second) {
+    return DateUtil.isValidDate(date, format) && DateTimeUtils.isValidTime(time, second);
```

Comment:
we usually try to name boolean flags such that it is obvious it's a boolean. We update to be something like `includeSeconds`, `hasSeconds`, etc. 

Also, these inline docs need to be updated to include the new parameter addition